storage/engine/keys.go

```diff
@@ -44,12 +45,17 @@ func StoreIdentKey() proto.Key {
 	return MakeStoreKey(KeyLocalStoreIdentSuffix, proto.Key{})
 }
 
-// StoreStatKey returns the key for accessing the named stat
-// for the specified store ID.
-func StoreStatKey(storeID int32, stat proto.Key) proto.Key {
+// StoreStatKey returns the key for accessing the named stat.
+func StoreStatKey(stat proto.Key) proto.Key {
 	return MakeStoreKey(KeyLocalStoreStatSuffix, stat)
 }
 
+// StoreStatusKey returns the key for accessing the store status for the
+// specified store ID.
+func StoreStatusKey(storeID int32) proto.Key {
+	return MakeKey(KeyStatusStorePrefix, proto.Key(strconv.FormatInt(int64(storeID), 10)))
```

Comment:
While you're at it, might be nice to move key generation from server/node.go for the store-idgen- key here and also use the EncodeUvarint.

I should mention specifically that we want to use the encoding because that will keep our stores sorted by store ID.